src/test_help/canned_histories.rs

```diff
@@ -567,6 +614,94 @@ pub fn cancel_scheduled_activity_with_signal_and_activity_task_cancel(
     t
 }
 
+/// 1: EVENT_TYPE_WORKFLOW_EXECUTION_STARTED
+/// 2: EVENT_TYPE_WORKFLOW_TASK_SCHEDULED
+/// 3: EVENT_TYPE_WORKFLOW_TASK_STARTED
+/// 4: EVENT_TYPE_WORKFLOW_TASK_COMPLETED
+/// 5: EVENT_TYPE_ACTIVITY_TASK_SCHEDULED
+/// 6: EVENT_TYPE_ACTIVITY_TASK_STARTED
+/// 7: EVENT_TYPE_WORKFLOW_EXECUTION_SIGNALED
+/// 8: EVENT_TYPE_WORKFLOW_TASK_SCHEDULED
+/// 9: EVENT_TYPE_WORKFLOW_TASK_STARTED
+/// 10: EVENT_TYPE_WORKFLOW_TASK_COMPLETED
+/// 11: EVENT_TYPE_ACTIVITY_TASK_CANCEL_REQUESTED
+/// 12: EVENT_TYPE_WORKFLOW_EXECUTION_SIGNALED
+/// 13: EVENT_TYPE_WORKFLOW_TASK_SCHEDULED
+/// 14: EVENT_TYPE_WORKFLOW_TASK_STARTED
+/// 15: EVENT_TYPE_WORKFLOW_TASK_COMPLETED
+/// 16: EVENT_TYPE_ACTIVITY_TASK_CANCELED
```

Comment:
I think maybe you don't need the signal here and can to AT_Canceled right after AT_Cancel_Requested and cover the same path? Not a big deal but might be able to shorten a bit.

Reply:
Oh, I guess that's the history of the other one you added. What different paths do they cover? Maybe worth adding a note about that.

Reply:
Which is scheduled vs started? If they cover identical paths maybe we don't need this signal version -- but it sounds like you're saying they cover different paths?

Reply:
They do cover different paths in the state machine due to activity started event.